### ISSUE TITLE
[Question] Allow everyone to use emojis without a permissions mod?

### ISSUE THREAD

**Issue by maxheyn** [Patbox/StyledChat#14]
Trying to set up the config on my op account. I would like everyone to be able to use the emojis in the list, but have no permissions mod. I have my config here below, was wondering if you could help me out and tell me what I am doing wrong.
Using the latest version on 1.18 server.

```json5
{
  "CONFIG_VERSION_DONT_TOUCH_THIS": 2,
  "_comment": "Before changing anything, see https://github.com/Patbox/StyledChat#configuration",
  "defaultStyle": {
    "displayName": "${name}",
    "chat": "<dark_green>✎</dark_green> <color:#3CE13F>${player} <gray>»</gray> <white>${message}",
    "join": "<dark_green>✚</dark_green> <color:#85ff8f><lang:multiplayer.player.joined:'${player}'>",
    "joinRenamed": "<dark_green>✚</dark_green> <color:#85ff8f><lang:multiplayer.player.joined.renamed:'${player}':'${old_name}'>",
    "left": "<dark_red>☁</dark_red> <color:#ff8585><lang:multiplayer.player.left:'${player}'>",
    "death": "<dark_red>☠</dark_red> <red>${default_message}...",
    "advancementTask": "<green>✔</green> <gold><lang:chat.type.advancement.task:'${player}':'${advancement}'>!",
    "advancementChallenge": "<dark_purple>✔</dark_purple> <gold><lang:chat.type.advancement.challenge:'${player}':'${advancement}'>!",
    "advancementGoal": "<green>✔</green> <gold><lang:chat.type.advancement.goal:'${player}':'${advancement}'>!",
    "teamChatSent": "<lang:'chat.type.team.sent':'<hover\\:\\'<lang\\:chat.type.team.hover>\\'><suggest_command\\:\\'/teammsg \\'>${team}':'${displayName}':'${message}'>",
    "teamChatReceived": "<lang:'chat.type.team.text':'<hover\\:\\'<lang\\:chat.type.team.hover>\\'><suggest_command\\:\\'/teammsg \\'>${team}':'${displayName}':'${message}'>",
    "privateMessageSent": "<color:#FF7EFF>To [${receiver}]: ${message}",
    "privateMessageReceived": "<color:#FF7EFF>[${sender}] whispers: ${message}",
    "sayCommand": "<color:#FEC1C0>[${player}] ${message}",
    "meCommand": "<color:#FF7E40><italic><lang:'chat.type.emote':'${player}':'${message}'>"
  },
  "petDeathMessage": "<dark_red>☠</dark_red> <red>${default_message}",
  "emoticons": {
    "potion": "🧪",
    "item": "[%player:equipment_slot/mainhand%]",
    "trident": "🔱",
    "rod": "🎣",
    "shrug": "¯\\_(ツ)_/¯",
    "bow": "🏹",
    "bell": "🔔",
    "heart": "❤",
    "bucket": "🪣",
    "sword": "🗡",
    "shears": "✂",
    "pos": "%player:pos_x% %player:pos_y% %player:pos_z%",
    "fire": "🔥",
    "table": "(╯°□°）╯︵ ┻━┻"
  },
  "legacyChatFormatting": true,
  "parseLinksInChat": true,
  "enableMarkdown": true,
  "formattingInPrivateMessages": true,
  "formattingInTeamMessages": true,
  "linkStyle": "<underline><blue>${link}",
  "spoilerStyle": "<gray>${spoiler}",
  "spoilerSymbol": "▌",
  "defaultEnabledFormatting": {
    "dark_red": true,
    "color": true,
    "underline": true,
    "yellow": true,
    "italic": true,
    "dark_blue": true,
    "dark_purple": true,
    "gold": true,
    "red": true,
    "aqua": true,
    "hover": true,
    "gray": true,
    "light_purple": true,
    "white": true,
    "dark_gray": true,
    "spoiler": true,
    "strikethrough": true,
    "lang": true,
    "obfuscated": true,
    "dark_grey": true,
    "key": true,
    "change_page": true,
    "st": true,
    "b": true,
    "green": true,
    "c": true,
    "dark_green": true,
    "gradient": true,
    "black": true,
    "em": true,
    "i": true,
    "bold": true,
    "gr": true,
    "grey": true,
    "orange": true,
    "rb": true,
    "rainbow": true,
    "obf": true,
    "colour": true,
    "blue": true,
    "dark_aqua": true,
    "underlined": true,
    "reset": true,
    "page": true,
    "font": true
  }
}
```

**Comment by maxheyn** [comment]
![image](https://user-images.githubusercontent.com/16215993/144122388-131accb7-6a20-4c40-99cf-944ea24344c8.png)

**Comment by Patbox** [comment]
Interesting, putting them in `emoticons` should work. I will look into that

**Comment by maxheyn** [comment]
> Interesting, putting them in `emoticons` should work. I will look into that

Sounds good, if you need any more information and want to get to me quickly then I am Maxypoo#4327 on the Fabric Discord.

**Comment by Patbox** [comment]
I can confirm the issue, I will release an fix in a moment

**Comment by maxheyn** [comment]
Just an update. I've set everything possible in the config to `true` and there are a few things that don't seem to be working at the moment. Fabric 1.18 server `0.12.6`.

Going based off of your image in the readme, I tried to replicate it.
![image](https://user-images.githubusercontent.com/16215993/144127448-06cc0c17-2b71-4ade-8b11-ed74272b2793.png)


![image](https://user-images.githubusercontent.com/16215993/144127425-9d8fac96-ee9c-43cb-9eab-eed2d700265f.png)

It appears that `||Spoilers||` aren't working, neither is `<item>` nor `:item:` (tried with both an item in and not in my hand).

I've updated the config in the original post to update what I have right now as of this comment.
